### PR TITLE
Add Admonitions to General Docs

### PR DESCRIPTION
**Pull request by salmad3**
Part of https://github.com/w3f/polkadot-wiki/issues/3133

### CODE REVIEW

File: docs/general/contributors.md [hunk 1]
```diff
@@ -13,11 +13,13 @@ is listed as a precaution - if one of these people ever contacts you directly, f
...
-:warning: **Employee impersonation is a common type of scam**. :warning:
+:::warning Employee impersonation is a common type of scam
 
-> For other ways of protecting yourself, please see [How to Protect Yourself from Scams](scams.md).
+For other ways of protecting yourself, please see the [How to Protect Yourself from Scams](scams.md) guide.
 
----
+:::
+
+## Technical Education
```
Comment (profradha) [inline docs/general/contributors.md]: The reorganization and changes to the Contributors page need to be in a different PR and will need separate review pipeline.

**Review by DrW3RK — CHANGES_REQUESTED** [review]
Admonitions are a very good idea. The admonitions on this page look good to me. Please move the changes to the contributors page to a different PR, as this PR's focus is admonitions and changes to contributor page might need additional review by the team and Joe

File: docs/general/contributors.md (hunk 1 above)
Comment (salmad3) [reply docs/general/contributors.md]: https://github.com/w3f/polkadot-wiki/pull/3168

**Review by DrW3RK — APPROVED** [review]
